src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-width;
+  width: calc($width-day * 7 + 10px * 2 + $gap-width * 6);
```

Comment:
Avoid using hard-coded values in calculations. Instead, you should define a variable for the padding and border size (10px and 1px respectively in this case) and use it in the calculation to maintain consistency and make it easier to change in the future.